environments/development/postgresql.yml

```diff
@@ -0,0 +1,11 @@
+DEFAULT_POSTGRESQL_PASSWORD: commcarehq
+DEFAULT_POSTGRESQL_USER: commcarehq
+SEPARATE_SYNCLOGS_DB: True
```

Comment:
If this is `True` surely there has to be a db in `postgresql_dbs` with a django alias of `synclogs`? I think this should be `False`. 

Also in travis and bootstrap config.

Seems worth validating. Maybe also worth making a constant out of it like `formplayer` and `commcarehq`

Reply:
Ah, yes, I got these backwards. Thanks!

Reply:
> Maybe also worth making a constant out of it like `formplayer` and `commcarehq`
I've got a little more refactoring to do here so that these are all treated more similarly in general, but that's a good point. I'll try and treat this constant names like the others.